scormxblock/static/sass/scormxblock.scss

```diff
@@ -0,0 +1,193 @@
+/* CSS for ScormXBlock */
+.close-navigation {
+    margin-bottom: 10px;
+
+    @media (max-width: 600px) {
+        margin: 0 0 30px 15px;
+    }
+
+    &:focus {
+        box-shadow: none;
+    }
+}
+
+.navigation-links {
+    text-align: center;
+}
+
+.container-fluid {
+    padding: 0 !important;
+}
+
+.full-screen-on {
+    display: block;
+    float: right;
+
+    @media (max-width: 600px) {
+        margin-right: 15px;
+    }
+}
+
+.scormxblock_block {
+    display: flex;
+    margin-bottom: 10px;
+
+    @media (max-width: 600px) {
+        flex-direction: column;
+    }
+
+    .js-scorm-block {
+        width: 100%;
+
+        @media (max-width: 600px) {
+            margin-bottom: 30px;
+        }
+    }
+
+    .scorm_object {
+        border: none;
+        margin-left: 15px;
+
+        @media (max-width: 600px) {
+            margin-left: 0;
+        }
+    }
+
+    span.setting-input-file {
+        padding-left: 25%;
+        margin-left: 14px;
+        padding-top: 5px;
+
+        span {
+            font-weight: bold;
+        }
+    }
+
+    .full-screen-scorm {
+        position: fixed;
+        border: none;
+        overflow: hidden;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        z-index: 9999999999;
```

Comment:
Why do you use a so high number of z-index?

Reply:
Done